src/main/java/duke/DialogBox.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
```

Comment:
Perhaps, you can separate the import statements of each type with a blank line (as shown in this guide [here](https://se-education.org/guides/conventions/java/intermediate.html))? You can do this in every class wherever you deem suitable!

Reply:
I agree, it would look more organised as well when they are segmented in an appropriate manner. I will take note and change it in the following iteration! Much appreciated!

Reply:
I have tried to add the changes, but apparently checkstyle is not happy with this 😂
<img width="394" alt="Screenshot 2023-02-05 at 7 36 19 AM" src="https://user-images.githubusercontent.com/50812868/216794032-b9d57d03-e44d-4002-9e41-528a8f70a3ff.png">
See here for checkstyle warning
<img width="756" alt="Screenshot 2023-02-05 at 7 36 28 AM" src="https://user-images.githubusercontent.com/50812868/216794054-b336fbdd-cb15-4b51-9f5a-41613ab505ed.png">
I believe that my initial ordering is fine because everything belongs to the same javafx package, then the sub packages are in alphabetical order. I suspect the checkstyle on really complains that the developer add a blank line is when there are 2 or more "master packages" like ```javafx``` and ```javax``` together.